nemo/core/config/modelPT.py

```diff
@@ -64,3 +63,53 @@ class NemoConfig:
     )
     exp_manager: Optional[Any] = exp_manager.ExpManagerConfig()
     hydra: HydraConfig = HydraConfig()
+
+
+class ModelConfigBuilder:
+    def __init__(self, model_cfg: ModelConfig):
```

Comment:
can you please add docstring to this class?

Reply:
Ah forgot about that one.

Reply:
Done